testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -784,7 +784,7 @@
                             <systemPropertyVariables>
                                 <keycloak.profile.feature.map_storage>enabled</keycloak.profile.feature.map_storage>
                                 <keycloak.mapStorage.provider>concurrenthashmap</keycloak.mapStorage.provider>
-                                <keycloak.globalLock.provider>none</keycloak.globalLock.provider>
+                                <keycloak.default.globalLock.provider>none</keycloak.default.globalLock.provider>
```

Comment:
Why is the `default` global lock provider needed?
Would it be possible to leverage provider order (`ProviderFactory.order`) instead?

Reply:
This could be solved by the `order`, however, this may not be necessary if https://github.com/keycloak/keycloak/pull/16080 is merged the way it is sent now.